### PR TITLE
DSDEEPB-2152 Changed bucket / group names to avoid word "rawls"

### PR DESCRIPTION
**Pull request by helgridly**
I'm opening a PR for this for three reasons:
1. Should this be hardcoded? Is it reasonable to assume that rawls = firecloud, or should it go in a config?
2. Should that string be `fcld` or something else?
3. Will this break existing workspaces that will now expect to be looked up with the string `fcld` when they're really named `rawls`?

---
- [x] **Submitter**: Rebase to develop. DO NOT SQUASH
- [x] **Submitter**: Make sure Swagger is updated if API changes
- [x] **Submitter**: Make sure documentation for code is complete
- [x] **Submitter**: Review code comments; remove done TODOs, create stories for remaining TODOs
- [x] **Submitter**: Include the JIRA issue number in the PR description
- [x] **Submitter**: Add description or comments on the PR explaining the hows/whys (if not obvious)
- [x] Tell  that the PR exists if he wants to look at it
- [x] Anoint a lead reviewer (LR). **Assign PR to LR**
- [x] **LR**: Initial review by LR and others.
- [x] Comment / review / update cycle:
  - Rest of team may comments on PR at will
  - **LR assigns to submitter** for feedback fixes
  - Submitter updates documentation as needed
  - Submitter rebases to develop again if necessary
  - Submitter makes further commits. DO NOT SQUASH. **Reassign to LR** for further feedback
- [x]  sign off
- [x] **LR** sign off
- [x] **Assign to submitter** to finalize
- [x] **Submitter**: Squash commits, rebase if necessary
- [x] **Submitter**: Verify all tests go green, including CI tests
- [x] **Submitter**: Merge to develop 
- [x] **Submitter**: Delete branch after merge
- [x] **Submitter**: Check configuration files in Jenkins in case they need changes
- [x] **Submitter**: Verify swagger UI on dev server still works after deployment
- [x] **Submitter**: Inform other teams of any API changes via hipchat and/or email
- [x] **Submitter**: Mark JIRA issue as resolved once this checklist is completed

### CODE REVIEW

File: src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/HttpGoogleServicesDAO.scala [hunk 1]
```diff
@@ -583,7 +583,7 @@ class HttpGoogleServicesDAO(
   def adminGroupName = s"${groupsPrefix}-ADMINS@${appsDomain}"
   def toGroupId(bucketName: String, accessLevel: WorkspaceAccessLevel) = s"${bucketName}-${accessLevel.toString}@${appsDomain}"
   def fromGroupId(groupId: String): Option[WorkspacePermissionsPair] = {
-    val pattern = s"rawls-([0-9a-f]+-[0-9a-f]+-[0-9a-f]+-[0-9a-f]+-[0-9a-f]+)-([a-zA-Z]+)@${appsDomain}".r
+    val pattern = s"fcld-([0-9a-f]+-[0-9a-f]+-[0-9a-f]+-[0-9a-f]+-[0-9a-f]+)-([a-zA-Z]+)@${appsDomain}".r
```
Comment (helgridly) [inline src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/HttpGoogleServicesDAO.scala]: Note I did groups as well.

**Comment by MatthewBemis** [comment]
regarding #3, I think existing workspaces will still be "usable" but you won't be able to delete them or do anything ACL related. afaik delete and ACLs are the only times we actually look up google groups once a workspace is created.

**Comment by dvoet** [comment]
I think it would be a good thing to make it configurable.

Also, the fact that deleting a workspace will stop working is bad. We store the bucket name for the workspace so we should use that instead of rebuilding the bucket name from the workspace id.

**Comment by helgridly** [comment]
Okay, updates:
- bucket name is now configurable (uses `groupsPrefix` already defined)
- delete workspace now uses the bucket name associated with the workspace rather than constructing it

What I _haven't_ done is replaced all calls to `getBucketName` with looking up the bucket from the workspace(id). Should I do that too? @biden @dvoet ?

**Comment by MatthewBemis** [comment]
re: What I haven't done is replaced all calls to getBucketName with looking up the bucket from the workspace(id). Should I do that too?

I would say yes for consistency

**Comment by helgridly** [comment]
That means a call to workspace DAO from googly land, which makes me uncomfortable. Or at least we have to pass that in.

**Comment by MatthewBemis** [comment]
I've changed my mind. I only said yes because if we change the prefix again, it would break the ties from workspace to google groups/buckets. If we have no plans to change the prefix, then it'll be fine. If we do change the prefix, then we could always go back and implement this to look up the bucketName from workspaceDAO to restore the ties to google, but doing so now totally isn't necessary for this PR imo.

**Comment by dvoet** [comment]
I leave it up to you whether or not to replace getBucketName calls now or not but I think it should be done. I would not do it from googly land but pass in the bucket name instead. If you choose not to do this, please enter a techdebt ticket. I agree it is not top priority.

otherwise :+1:

**Comment by helgridly** [comment]
Added techdebt ticket in DSDEEPB-2468.

**Pull request merged**